first-read-last-write-cache/src/access.rs

```diff
@@ -248,4 +248,115 @@ mod tests {
             }
         )
     }
+
+    #[test]
+    fn test_err_merge_left_read_neq_right_read() {
+        let first_read = 1;
+        let mut value = create_value(first_read);
```

Comment:
`mut` is not needed for any of the variables in this test (the same for other tests).